### ISSUE TITLE
Pretrained Weights

### ISSUE THREAD

**Issue by koustav123** [shedy-pub/hlagcn-jittor#2]
Hi,

Are there any pre-trained weights available for AADB dataset?

I tried to run the training on AADB dataset. I get this warning both during training and testing. And my numbers are **0.66/0.63** on AADB, much lower than reported in the paper (**0.90/0.89**). I did not change anything else in the code.

Also, I am running the code on a single GPU in a virtual environment on a single PC without mpi. 

![image](https://user-images.githubusercontent.com/3684879/148771150-a9919a38-add7-49f8-9ce7-681485124f24.png)

Any pointers will be highly helpful. 

Thanks

**Comment by evenhax** [comment]
Got the same problem, looking forward to your reply.

**Comment by islandLZ** [comment]
Hello, excuse me, MSE, SRCC, LCC, ACC, EMD_ 1 、EMD_2 ，What do these represent respectively? And how to calculate the score?

**Comment by flyingbird93** [comment]
Have you ever encountered a situation where the loss output is always "nan"? 
If so how did you solve it?

Do you ensure proper use of train, validation and test sets during training?
Probably this is why the results are poor on AADB.

**Comment by dafeigediaozhatian** [comment]
I suggest that we can set up a WeChat group to discuss the reason for the code, or send an email to the author to ask the bad results.
